docs/how-to/evaluate/custom_metric.md

```diff
@@ -0,0 +1,7 @@
+# How to create custom Metric for Ragbits evaluation
+
+`ragbits.evaluate` package provides the implementation of metrics that measure the quality of document search pipeline within `ragbits.evaluate.metrics.document_search`
```

Comment:
I think we should have an index of all metrics predefined in ragbits, with each of them described.